ctakesclient/resources/covid_symptoms.bsv

```diff
@@ -143,7 +143,6 @@ C0003126|T033|44169009|SNOMEDCT_US|Lost the sense of smell|Loss of taste or smel
 C0003126|T033|44169009|SNOMEDCT_US|Lost smell|Loss of taste or smell
 C0003126|T033|44169009|SNOMEDCT_US|No sense of smell|Loss of taste or smell
 C0003126|T033|44169009|SNOMEDCT_US|could not smell|Loss of taste or smell
-C0003126|T033|R43|ICD10CM|R43|Loss of taste or smell
```

Comment:
Why delete this line?

Reply:
Because it is wrong / sloppy. 
For NLP it is correct but for real ICD10 it is wrong. 
For NLP it is duplicative - in the ED note it was mentioned as "R43 Anosmia". 
We already have a match for Anosmia , and R43 is just plain wrong because that could be a loss of taste (Aguesia)